requirements.txt

```diff
@@ -1,3 +1,4 @@
 click>=6.6
 # PyYAML
 six>=1.10
+requests
```

Comment:
Can this set a minimum version?

Reply:
no idea, any suggestions?

Reply:
i dont think the api changed where i use it so I dont really have any minimum in mind

Reply:
Just set the minimum to the latest version with `requests>=2.25.1`.

Reply:
done